s3bench.py

```diff
@@ -55,6 +58,7 @@ def __init__(self):
         self.object_name = ""
         self.num_objects = args.num_objects
         self.workload = args.workload
+        self.max_latency = args.max_latency
```

Comment:
1. We should use a ternary if the condition for the `self.max_latency` value 

Why? 
1. If the customer didn't use the `--max-latency` flag, we should set the value to 0 (if there is no max latency provided, all requests are in the correct range. 

```suggestion 
self.max_latency = args.max_latency if args.max_latency else 0
```

Reply:
agreed